pkg/argocd/update.go

```diff
@@ -288,6 +288,18 @@ func UpdateApplication(updateConf *UpdateConfiguration, state *SyncIterationStat
 		if needsUpdate(updateableImage, applicationImage, latest) {
 
 			imgCtx.Infof("Setting new image to %s", applicationImage.WithTag(latest).GetFullNameWithTag())
```

Comment:
can we move this log line a few lines below around line 304, after `needUpdate = true`?

Reply:
done